components/fieldlabel/stories/form.template.js

```diff
@@ -43,21 +43,11 @@ export const Template = ({
                         alignment: labelPosition === "side" ? fieldLabelAlignment : undefined,
                     }, context))}
                     <div class=${classMap({
-                        [`${rootClass}-item`]: true,
+                        [`${rootClass}-itemField`]: true,
                     })}>
-                        ${when(label, () => FieldLabel({
```

Comment:
This is a great update but feels out of scope for this work. Would you mind opening a PR for this off main?

Reply:
Good call, this has been dropped. 

I only see this duplicate code on the s2-foundations branches. Right now, I have a draft PR, but it's against `s2-foundations-design-links`. I'll rebase and get that ready for `s2-foundations-redux`.